ledger/sandbox/src/main/resources/db/migration/postgres/V20__Events_new_schema.sql

```diff
@@ -0,0 +1,76 @@
+-- Copyright (c) 2019 The DAML Authors. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+-- contains all events for all transactions
+create table participant_events
+(
+    event_id varchar primary key not null,
+    event_offset bigint not null,
+    contract_id varchar not null,
+    transaction_id varchar not null,
+    ledger_effective_time timestamp not null,
+    template_package_id varchar not null,
+    template_name varchar not null,
+    node_index int not null,               -- post-traversal order of an event within a transaction
+    is_root boolean not null,
+
+    -- these fields can be null if the transaction originated in another participant
+    command_id varchar,
+    workflow_id varchar,                   -- null unless provided by a Ledger API call
+    application_id varchar,
+    submitter varchar,
+
+    -- non-null iff this event is a create
+    create_argument bytea,
+    create_signatories varchar array,
+    create_observers varchar array,
+    create_agreement_text varchar,         -- null if agreement text is not provided
+    create_consumed_at varchar,            -- null if the contract created by this event is active
+    key_template_package_id varchar,       -- null if the contract created by this event has no key
+    key_template_name varchar,             -- null if the contract created by this event has no key
+    key_value bytea,                       -- null if the contract created by this event has no key
```

Comment:
Why are these not prefixed with `create_`?

Reply:
Originally in the proposal they were supposed to go into their own table and I thought it was unnecessary do add them afterwards, but I can add it.

Reply:
More importantly, I just noticed that only `key_value` should be kept, the others should go away.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/5029/commits/580296f46e11bdbf48ffee6d6071962630781dae